bindings/node/native/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "node-reconfix"
+version = "0.1.0"
+authors = ["Aaron Brodersen <aaron@resin.io>"]
+license = "MIT"
```

Comment:
Out of curiosity, how come this is MIT rather than Apache-2.0 ?

Reply:
@lurch I think Neon generated this and defaults to MIT.